Makefile

```diff
@@ -41,4 +43,23 @@ lint:
 install:
 	for d in $(SUBDIRS); do $(MAKE) -C $$d install; done
 
-.PHONY: all $(SUBDIRS) clean proto deps lint install
+sandbox-test-cri-build: sandbox-test-cri-build-stamp
```

Comment:
nit: I would probably put all of these above `clean` because I tend to like that toward the bottom.  But I don't have a very strong opinion here.  @nmeyerhans do you have a preference?

Reply:
I moved it down as after pointing it out I prefer that as well.